packages/next/src/client/components/app-router.tsx

```diff
@@ -149,7 +149,7 @@ function HistoryUpdater({
 
 export const createEmptyCacheNode = () => ({
   lazyData: null,
-  subTreeData: null,
+  rsc: null,
```

Comment:
Do we consider to call it sth like `rscNode` or any related to "node" as the type is ReactNode, rsc makes me think of rsc payload data

Reply:
Yeah maybe but even `rscNode` could refer to something that was rendered by RSC but is not renderable on the client. It would have to be something more like `rscReactNode` but I don't know if that really does much to clarify. I think in this case we can rely on the type to communicate it.